Sources/COWMacros/SyntaxExtensions.swift

```diff
@@ -621,19 +765,49 @@ extension SwitchCaseSyntax {
   
 }
 
-private let autoSynthesizingProtocolTypes: Set<String> = [
+let equatableProtocolNames: Set<String> = [
   "Equatable",
-  "Swift.Equatable",
+  "Swift.Equatable"
+]
+
+let hashableProtocolNames: Set<String> = [
   "Hashable",
-  "Swift.Hashable",
+  "Swift.Hashable"
+]
+
+let codableProtocolNames: Set<String> = [
   "Codable",
-  "Swift.Codable",
+  "Swift.Codable"
+]
+
+let encodableProtocolNames: Set<String> = [
```

Comment:
Explicit access control is recommended. Use `private` or `internal`. This convention is learned from the Swift Standard Library.